types.go

```diff
@@ -19,8 +19,8 @@ type Prefix []byte
 // hash from the one used for interior nodes.
 type KeyValuePair struct {
 	_struct bool        `codec:",toarray"`
-	Key     Hash        `codec:"key"`
-	Value   interface{} `codec:"value"`
+	Key     Hash        `codec:"k"`
+	Value   interface{} `codec:"v"`
```

Comment:
I'm wondering if maybe making the `Value` a byte array is simpler? I don't have it written this way right now but I was thinking of converting my values to byte arrays-- for public nodes I have a clearly defined struct but for private nodes I just have a blob of encrypted data. I also already know the types because the folder IDs (my `Key`s) contain that information.